webpack.config.prod.js

```diff
@@ -0,0 +1,55 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  mode: 'production',
+  entry: './amt/site/static/ts/amt.ts',
+  output: {
+    filename: 'amt.[contenthash].js',
+    path: path.resolve(__dirname, 'amt/site/static/dist'),
```

Comment:
I read it is wise to add 'clean: true' so the dist folders is cleared before building.

```suggestion
    path: path.resolve(__dirname, 'amt/site/static/dist'),
    clean: true
```